### PR TITLE
Added new icons + modified old icons

### PR DESCRIPTION
**Pull request by odmln**
Added 40+ new icons and made changes to a few old icons.

### CODE REVIEW

File: icons/new/attention.svg [hunk 1]
```diff
@@ -1,9 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (jxnblk) [inline icons/new/attention.svg]: If anyone was using this icon, this would mean a breaking change – did you rename this one?

Comment (odmln) [reply icons/new/attention.svg]: I think I removed it by accident :/

File: icons/new/cruise.svg [hunk 2]
```diff
@@ -1,12 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (hakimelek) [inline icons/new/cruise.svg]: @odmln should we put this back?

File: icons/new/zoom out.svg [hunk 3]
```diff
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="utf-8"?>
```
Comment (hakimelek) [inline icons/new/zoom out.svg]: Same for this one

Comment (hakimelek) [reply icons/new/zoom out.svg]: Renamed to zoomOut

File: icons/new/cruise.svg (hunk 2 above)
Comment (hakimelek) [reply icons/new/cruise.svg]: Renamed to cruises.svg

**Pull request merged**